includes/class-wc-gateway-stripe.php

```diff
@@ -647,4 +650,42 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 			);
 		}
 	}
+
+	/**
+	 * Displays the Stripe fee
+	 * @param $order_id
+	 */
+	public function display_order_fee( $order_id ) {
+		$fee = get_post_meta( $order_id, self::META_NAME_FEE, true );
```

Comment:
Needs a CRUD way of getting order meta with BW compat logic.

Reply:
I will pass for a beast, but I do not understand well or you want to come, especially on the BW compat

Reply:
You can see how we're doing it here https://github.com/woocommerce/woocommerce-gateway-stripe/blob/master/includes/abstracts/abstract-wc-stripe-payment-gateway.php#L144-L147

Reply:
ok so I understood. On the other hand the question does not arise, because the element sent is not an object but just the ID of the order

https://github.com/woocommerce/woocommerce/blob/master/includes/admin/meta-boxes/views/html-order-items.php#L194